ubi_manifest/worker/tasks/depsolver/rpm_depsolver.py

```diff
@@ -152,6 +162,26 @@ def extract_and_resolve(self, content: set[UbiUnit]) -> None:
                     solved.add(req)
             self._unsolved -= solved
 
+    def resolve_files(
+        self, repos: list[YumRepository], blacklist: list[PackageToExclude]
+    ):
+        """
+        Get RPMs containing required files
+        """
+        content = set(self._file_req_rpms)
```

Comment:
As I stated elsewhere I don't get why you use `self._file_req_rpms` , I think the adding this to output of function is useless.